Bottom: de67a56768f7d510781e7af66bef74408135a9d8
Top:    058204bfa0e8eabd20f79fe43b90b0f042529437
Author: scott snyder <sss@karma>
Date:   2018-08-26 20:24:30 -0400

Bump target version to 20.



---

diff --git a/ComicReader/project.properties b/ComicReader/project.properties
index ea89160..92b1625 100755
--- a/ComicReader/project.properties
+++ b/ComicReader/project.properties
@@ -8,4 +8,4 @@
 # project structure.
 
 # Project target.
-target=android-8
+target=android-20
